src/darwin/Framework/CHIP/MTRDeviceController.mm

```diff
@@ -548,20 +548,33 @@ - (void)setDeviceControllerDelegate:(id<MTRDeviceControllerDelegate>)delegate qu
     });
 }
 
-- (void)setNocChainIssuer:(id<MTRNOCChainIssuer>)nocChainIssuer queue:(dispatch_queue_t)queue
+- (BOOL)setOperationalCertificateIssuer:(nullable id<MTROperationalCertificateIssuer>)operationalCertificateIssuer
```

Comment:
rather than pass back a bool, can we just pass back the error code?

In this implementation, error is sometimes set and sometimes unset and its an out var, might be just simpler to pass back `CHIP_NO_ERROR` if its a success

Reply:
This is the normal API shape for sync-fallible Darwin APIs: a return type that can be tested as a boolean, and an out param for the error in case the consumer wants it.